packs/elastic/README.md

```diff
@@ -0,0 +1,170 @@
+# Elasticsearch integration pack
+
+Pack provides many operations helping to manage your Elasticsearch indices. Current functionality is mostly based on [curator](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/).
+
+## Features
+
+Pack provides a set of actions based on curator.
+
+## Curator based actions
+
+Action | Description
+------ | -----------
+**alias** | Add indices to or remove them from aliases.
+**allocation** | Set routing allocation based on tags.
+**bloom** | Disable the bloom filter cache for indices.
+**close** | Close indices.
+**delete.indices** | Delete indices.
+**delete.snapshots** | Delete snapshots.
+**open** | Open indices.
```

Comment:
Yes, but `delete` can also operate on other resources (e.g. a document). 

And it's not just about explicitness, it's also about the consistency - all names having prefix vs some having prefix (on top of that, some other packs already follow this convention - `st2.kv.get`, `st2.actions.list`, etc.).

And using resource as a prefix means you can easily use prefix filter match to find the actions for a particular resource.